### PR TITLE
GLX backporting related fixes

### PR DESCRIPTION
**Pull request by uli42**
just some small fixes I found during my Mesa upgrade work.

### CODE REVIEW

File: nx-X11/programs/Xserver/GL/glx/glxext.c [hunk 1]
```diff
@@ -246,7 +245,7 @@ void GlxExtensionInit(void)
     */
     extEntry = AddExtension(GLX_EXTENSION_NAME, __GLX_NUMBER_EVENTS,
 			    __GLX_NUMBER_ERRORS, __glXDispatch,
-			    __glXSwapDispatch, ResetExtension,
+			    __glXDispatch, ResetExtension,
```
Comment (sunweaver) [inline nx-X11/programs/Xserver/GL/glx/glxext.c:248]: You are listing _glxDispatch twice here, now.

**Review by sunweaver — CHANGES_REQUESTED** [review]
@uli42: one minor fix required for PR#532. Thanks for working on this!

**Comment by uli42** [comment]
Yes, that's on purpose as the function is used for normal _and_ the swapped case.

Vom Smartphone gesendet.

----- Ursprüngliche Nachricht -----
Von: "Mike Gabriel" <notifications@github.com>
Gesendet: ‎21.‎10.‎2017 10:35
An: "ArcticaProject/nx-libs" <nx-libs@noreply.github.com>
Cc: "Ulrich Sibiller" <uli42@gmx.de>; "Mention" <mention@noreply.github.com>
Betreff: Re: [ArcticaProject/nx-libs] Small fixes (#532)

@sunweaver requested changes on this pull request.
@uli42: one minor fix required for PR#532. Thanks for working on this!



In nx-X11/programs/Xserver/GL/glx/glxext.c:
> @@ -246,7 +245,7 @@ void GlxExtensionInit(void)
     */
     extEntry = AddExtension(GLX_EXTENSION_NAME, __GLX_NUMBER_EVENTS,
 			    __GLX_NUMBER_ERRORS, __glXDispatch,
-			    __glXSwapDispatch, ResetExtension,
+			    __glXDispatch, ResetExtension,
You are listing _glxDispatch twice here, now.

**Comment by sunweaver** [comment]
On  Sa 21 Okt 2017 11:22:30 CEST, Ulrich Sibiller wrote:

> Yes, that's on purpose as the function is used for normal _and_ the  
> swapped case.

Ah. Ok.

Let me know if you have more commits for this branch (see one of my  
other comments today). Ping me via Github, when the small_fixes branch  
is ready to merge.

_o/
Mike
-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

**Comment by sunweaver** [comment]
@uli42: can you please rebase this branch against 3.6.x?

**Review by sunweaver — APPROVED** [review]

**Pull request merged**